Prompt install if first action is run dev mode

If you immediately run `acorn run -i .` you will get the following message:

```
ERRO[0000] Failed to build /Users/darren/src/examples/guestbook/Acornfile: no matches for kind "Builder" in version "api.acorn.io/v1" 
```
We should prompt the user if they want to install like we do with `acorn run` in non-dev mode.

Found nil pointer deference 
```
? Acorn is not installed, do you want to install it now?: Yes
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x19229b1]

goroutine 1 [running]:
github.com/acorn-io/acorn/pkg/install.Install({0x2f14378, 0xc0005670c0}, {0xc00064a160, 0x1b}, 0x0)
	/home/acorn/go/src/github.com/meldafrawi/acorn/pkg/install/install.go:120 +0xb1
github.com/acorn-io/acorn/pkg/client.promptInstall[...]({0x2f14378?, 0xc0005670c0}, 0xc00108f600)
	/home/acorn/go/src/github.com/meldafrawi/acorn/pkg/client/ignore.go:38 +0x16b
github.com/acorn-io/acorn/pkg/client.IgnoreUninstalled.Info({{0x2f3e7a8?, 0xc0006bca40?}}, {0x2f14378?, 0xc0005670c0?})
	/home/acorn/go/src/github.com/meldafrawi/acorn/pkg/client/ignore.go:251 +0x71
github.com/acorn-io/acorn/pkg/cli.(*Run).Run(0xc00057cb00, 0xc000911b80, {0xc00083e300?, 0x1, 0x2})
	/home/acorn/go/src/github.com/meldafrawi/acorn/pkg/cli/run.go:191 +0xbe
github.com/acorn-io/acorn/pkg/cli/builder.bind.func1(0xc000911b80?, {0xc00083e300, 0x1, 0x2})
	/home/acorn/go/src/github.com/meldafrawi/acorn/pkg/cli/builder/builder.go:355 +0x19d
github.com/spf13/cobra.(*Command).execute(0xc000911b80, {0xc00083e2e0, 0x2, 0x2})
	/home/acorn/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:872 +0x694
github.com/spf13/cobra.(*Command).ExecuteC(0xc00077f680)
	/home/acorn/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	/home/acorn/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	/home/acorn/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:911
main.main()
	/home/acorn/go/src/github.com/meldafrawi/acorn/main.go:20 +0x126
exit status 2
```

--- a/pkg/client/ignore.go
+++ b/pkg/client/ignore.go
@@ -35,7 +35,7 @@ func promptInstall[V any](ctx context.Context, f twoFunc[V]) (V, error) {
 		}
 
 		if shouldInstall {
-			installErr := install.Install(ctx, install.DefaultImage(), nil)
+			installErr := install.Install(ctx, install.DefaultImage(), &install.Options{})
 			if installErr != nil {
 				return v, installErr
 			}

